packages/@react-spectrum/s2/intl/it-IT.json

```diff
@@ -7,12 +8,12 @@
   "dropzone.replaceMessage": "Rilascia il file da sostituire",
   "inlinealert.informative": "Informazioni",
   "inlinealert.negative": "Errore",
-  "inlinealert.notice": "Avviso",
+  "inlinealert.notice": "Avvertenza",
```

Comment:
Same - for `packages/@react-spectrum/inlinealert/intl/it-IT.json`

Reply:
I've updated to "Alert" are consistent and "Warning" are consistent, and will do a followup review with vendor